uktrade/directory-cms
# KLS-900 - Use wagtailmarkdown field instead of custom markdown field

This PR fixes a bug in which markdown fields in wagtail admin do not show the content of the field until you click into it.
The project currently uses a custom Markdown field which relies on javascript copied from the [simple-markdown-editor package](https://github.com/sparksuite/simplemde-markdown-editor) (not updated since 2016), and it's this javascript which is causing the bug.

This PR introduces a new package - [wagtail-markdown](https://github.com/torchbox/wagtail-markdown) which has more active development, and replaces all uses of the custom markdown field with the one from the package.

**Removal of custom markdown field**
There will be a PR to follow which removes the custom markdown field. Because this will include updating many historic migrations, I have separated the two PRs.

**To test**
- Run `make manage migrate`
- Go to a page in admin which has a markdown field - e.g. the article body text field here - http://cms.trade.great:8010/admin/pages/770/edit/
- Markdown should appear as normal in the field
- The live page should remain unchanged

**Screenshot - before**
<img width="884" alt="Screenshot 2023-07-20 at 12 05 56" src="https://github.com/uktrade/directory-cms/assets/22460823/43fda7a8-80ac-4cf7-9e92-9d61885fc5f9">


**Screenshot - after**
<img width="855" alt="Screenshot 2023-07-20 at 12 06 14" src="https://github.com/uktrade/directory-cms/assets/22460823/57fe1a89-087f-4579-bb7b-e44b57cfb04c">

### Workflow

- [x] Ticket exists in Jira https://uktrade.atlassian.net/browse/KLS-900

### Reviewing help


### Housekeeping


### Merging
